### PR TITLE
AdAway revamped website

### PR DESCRIPTION
**Pull request by AnXh3L0**
Hello,

I have been an AdAway user since many years now and wanted to contribute to it. Seeing as I'm not a good Android dev, I decided to put my hands on the website and revamped it a bit. Below you'll see a screenshot of the entire new page.

I used Jekyll as it's officially supported by GitHub Pages and it's one of the easiest static website generators to use. I'd be pleased if this gets merged and if there's changes you'd like to make I can gladly implement them.

Thank you in advance!

image

### CODE REVIEW

**Comment by PerfectSlayer** [comment]
First, thanks for your contribution!  
The site definitely needs an upgrade and it is awesome you did it. 🎉

It is a major contribution to the project as I am not an expert in front development and I don't really have the time to maintain it. I never use Jekyll but it is not an issue to me to use it. 
The main design looks good and is definitely better than the current one so I will merge the PR.

Just few questions before:
- I see you mainly copy the content site content to a new design. With the new version coming (in some months), are you also interested to update the site with its new features? It will be mainly capture, app requirements, stuff like that. It is not a huge work, it's more to get an idea if you would consider your contribution as a one shot or if you are interested to keep the site running? (even a yearly content update will be great!)
- Is it possible to get the app title bigger? Something similar to the headline _Ad-blocking for your Android_ because the top part feels empty.
- Why do you remove `CNAME` file? Should we keep it to get `adaway.org` domain working?
- Is there some third party Open Source licenses to include to the site or the repository?

Thanks again for your work! 🙏

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-adaway.org
```
Comment (PerfectSlayer) [inline CNAME]: Isn't this configuration needed?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,23 @@
+The MIT License (MIT)
```
Comment (PerfectSlayer) [inline LICENSE]: The application code base is GPL.
Is there a reason to use MIT over GPL?

File: CNAME (hunk 1 above)
Comment (dschuermann) [reply CNAME]: Yep it's needed if you still want to use the domain. btw: @PerfectSlayer if you like to take over the domain, email me.

**Comment by AnXh3L0** [comment]
Hi, 

I'm on mobile at the moment so I'll keep it short but I'll write a longer comment as soon as I'm in front of the PC.

First of all, thank you for accepting my contribution. I can gladly help maintain the website in the long run, so you can count on me for the updates.
The CNAME is needed but I removed it momentarily to get the page working on my account with GitHub Pages. I can readd it and fix stuff in the next commits. 
The license is MIT because I created the theme/template a while back for the agency I'm working at, so the theme itself is MIT but the contents can be whatever open license you wish.

**Comment by AnXh3L0** [comment]
I made the required changes @PerfectSlayer , below's the new screenshot of the page.

image

**Comment by PerfectSlayer** [comment]
Awesome! Thanks for the quick update ;)

About licensing, does it bother you if we kept the concent of the site under GPL license in the `LICENSE.md` file? The theme could stay under MIT, you can even put your name in the footer if you want to.
Just split the file in two sections like the app repo does: https://github.com/AdAway/AdAway/blob/master/LICENSE.md

Otherwise, I am ready to merge and publish :)

**Comment by AnXh3L0** [comment]
Sure, we can keep content as GPL. I will make another commit for the README later. 

Is it an issue if you add me as a contributor with push access to the repo for further updates?

**Comment by PerfectSlayer** [comment]
Sure! Keep in mind it contains the hosts.txt which is included in the default AdAway configuration.
So try to not mess with it (even if GitHub will keep an history).

I just created a new branch to update the README and LICENSE file and test it. I will merge it to the master tonight 😉  And if everything works as expected, we should have a beautiful new website! 🎉  Thanks again for your work.

**Pull request merged**